program/print-diamond-pattern/PrintDiamondPattern.js

```diff
@@ -0,0 +1,12 @@
+const diamondPattern = (n) => {
+   let total = n, iteration = 0;
+   const construct = ['*'.repeat(n)];
+   while((n-=2) > 0) {
+      const layer = ' '.repeat(++iteration)+'*'.repeat(n);
```

Comment:
Change to this, so it'll work as shown in #4674

```suggestion
   let iteration = 0;
   const construct = ['*'.repeat(2 * n - 1)];
   n *= 2;
   while ((n -= 2) > 0) {
      const layer = ' '.repeat(++iteration) + '*'.repeat(n - 1);
```